src/server/test/apis/webhook-test.ts

```diff
@@ -17,7 +17,7 @@ const TEST_SECRETS = {
 test.before(() => {
   // See https://cloud.google.com/docs/authentication/production
   if (!process.env.GOOGLE_APPLICATION_CREDENTIALS) {
-    const rootDir = path.join(__dirname, '..', '..', '..');
+    const rootDir = path.join(__dirname, '..', '..', '..', '..');
```

Comment:
`path.resolve(__dirname, '../../../../')` seems clearer to me.

Reply:
Double check if that works on windows though?

Reply:
It won't work on Windows - hence this.